frontend/server/src/Controllers/Contest.php

```diff
@@ -3223,22 +3209,27 @@ private static function validateClarifications(\OmegaUp\Request $r): \OmegaUp\DA
      */
     public static function apiClarifications(\OmegaUp\Request $r): array {
         $r->ensureIdentity();
-        $r->ensureOptionalInt('offset' /* optional */);
-        $r->ensureOptionalInt('rowcount' /* optional */);
-        $contest = self::validateClarifications($r);
+        $offset = $r->ensureOptionalInt('offset') ?? null;
```

Comment:
```suggestion
        $offset = $r->ensureOptionalInt('offset');
```

esto es equivalente: `null ?? null === null`!

Reply:
Listo.